src/disco/containers/DiscoPane.js

```diff
@@ -67,6 +67,7 @@ class DiscoPane extends React.Component {
           </div>
         </header>
         {results.map((item, i) => <Addon {...camelCaseProps(item)} key={i} />)}
+        <a className="cta" href="https://addons.mozilla.org/">{i18n.gettext('See more add-ons!')}</a>
```

Comment:
I wrote this up yesterday but forget to submit it.

I'm starting to get concerned about our CSS. We are reusing some classes like and `cta` looks like it will probably get reused without whoever uses it knowing this style exists. We have some styles on just `header` and `header h1` too which seems fragile. There are also some styles that are nested 4 levels deep that could likely be reduced to 1 or 2 if we had more specific class names.

I know we discussed using some sort of methodology before and didn't settle on anything but I think we should try and come up with something. London might be a good place to discuss this.